packages/server/modules/cli/commands/workspaces/set-plan.ts

```diff
@@ -52,6 +52,7 @@ const command: CommandModule<
 
     await upsertPaidWorkspacePlanFactory({ db })({
       workspacePlan: {
+        createdAt: new Date(),
```

Comment:
I'd have created a type that makes createdAt optional to avoid having to set it everywhere, but this works